analyzer/src/main/kotlin/managers/Conda.kt

```diff
@@ -0,0 +1,162 @@
+        }
+
+        val result = support.resolveDependencies(definitionFile)
+
+        // Remove the env by simply deleting the directory.
+        envDir.safeDeleteRecursively()
+
+        return result
+    }
+
+    private fun setupEnv(definitionFile: File): File {
+        // Create an out-of-tree environment.
+        log.info { "Creating a Conda env for $definitionFile..." }
+        val envDir = createEnv(definitionFile)
+        val install = installDependencies(definitionFile, envDir)
+
+        if (install.isError) {
+            log.debug {
+                // Pip writes the real error message to stdout instead of stderr.
+                "First try to install dependencies using Conda failed with:\n${install.stdout}"
+            }
+
+            if (install.isError) {
+                // pip writes the real error message to stdout instead of stderr.
+                throw IOException("setupEnv (in $envDir) error: ${install.stdout}")
+            }
+        }
+
+        log.info {
+            "Successfully installed dependencies for project '$definitionFile' using Conda."
+        }
+
+        return envDir
+    }
+
+    /**
+     * Create a Conda environment given the environment.yml or requirements.txt.
+     */
+    fun createEnv(envDefinition: File): File {
+        if (envDefinition.name.endsWith(".py")) {
+            // Python install script is not an environment definition.
+            ProcessCapture(
+                "conda", "create",
+                "-y",
+                "--name", envName,
+                "python"
+            ).requireSuccess()
+        } else {
+            ProcessCapture(
+                "conda", "env", "create",
+                "--force",
+                "--name", envName,
+                "--file", envDefinition.absolutePath
+            ).requireSuccess()
+        }
+
+        // Load the list of Conda environments and find the one we created.
+        val jsonString: String = ProcessCapture("conda", "env", "list", "--json").requireSuccess().stdout
+        val rootNode = jsonMapper.readTree(jsonString)
+        val envsNodes = rootNode.path("envs").elements().asSequence().toList()
+        return File(envsNodes.single { File(it.textValue()).name == envName }.textValue())
+    }
+
+    private fun installDependencies(definitionFile: File, envDir: File): ProcessCapture {
+        // Install pipdeptree inside the Conda environment as that is the only way to make it report only the project's
+        // dependencies instead of those of all (globally) installed packages, see
+        // https://github.com/naiquevin/pipdeptree#known-issues.
+        // We only depend on pipdeptree to be at least version 0.5.0 for JSON output, but we stick to a fixed
+        // version to be sure to get consistent results.
+        runInEnv(envDir, "pip", *Pip.TRUSTED_HOSTS, "install", "pipdeptree==$PIPDEPTREE_VERSION").requireSuccess()
+
+        // Use the yml that defines the environment.
```

Comment:
The comment in line `151` is confusing me a bit.
Would it harm if it was actually removed?